Cubical/Categories/Displayed/TotalCategory/Base.agda

```diff
@@ -0,0 +1,90 @@
+{-# OPTIONS --safe #-}
+module Cubical.Categories.Displayed.TotalCategory.Base where
```

Comment:
Maybe a radical suggestion but I think we should put this in `Categories.Constructions.TotalCategory` (not under `Displayed`) because it constructs a `Category`, not a displayed category. Then `DisplayedTotalCategory` would go in `Categories.Displayed.Constructions.TotalCategory`.